charts/newrelic-infrastructure/values.yaml

```diff
@@ -188,6 +207,10 @@ controlPlane:
   # -- Run Control Plane scraper with `hostNetwork`.
   # `hostNetwork` is required for most control plane configurations, as they only accept connections from localhost.
   hostNetwork: true
+  # -- Config for the Infrastructure agent.
+  # Will be used by the forwarder sidecars and the agent running integrations.
```

Comment:
This one is correct but I think message should be more specific so users can tell the difference between this one and the `common` one.

Reply:
I changed the wording in the comit [#34a64bf](https://github.com/newrelic/nri-kubernetes/pull/422/commits/34a64bfdba0341b3f159726705bcf0e54541c9f4)

LMKWYT